src/unicode_utils.cc

```diff
@@ -6,7 +6,7 @@
 #include <cstddef>
 #include <cstdint>
 
-#ifdef _JSRS_USE_FULL_TABLES_
+#ifdef _JSTP_USE_FULL_TABLES_
```

Comment:
I'd like it to be `_JSTP_USE_FULL_TABLES_`.
P.S.: I have already asked you about this define, see: https://github.com/metarhia/jstp/pull/229#discussion_r126773168.